### PR TITLE
add setup documentation

### PR DESCRIPTION
**Pull request by cgundy**
I'm still having permission issues, so I wasn't able to push directly to this repo.

I'm new to Ruby, so please comment with any other suggestions or tips for setup. I ran into several issues, so I added a section on troubleshooting in case anyone ran into these same issues. But if you think any of these things are too obvious and clutter the README, I'm happy to remove them.

### CODE REVIEW

**Review by daleinen7 — APPROVED** [review]
This looks great to me. The only other idea I'd have is maybe to move the trouble shooting to the bottom? But I definitely think it should be there, and it didn't read as cluttered to me.

**Comment by cgundy** [comment]
> This looks great to me. The only other idea I'd have is maybe to move the trouble shooting to the bottom? But I definitely think it should be there, and it didn't read as cluttered to me.

Thanks! It might be hard to tell with how github displays changes, but they are at the bottom. 👍

**Comment by aaronreba** [comment]
Does `brew` also start up a database instance for postgres related commands? I can make a small writeup on using the postgres commands directly for starting a dedicated project-specific instance for anyone who has installed postgres using yum/apt or compiled postgres from source.

File: README.md [hunk 1]
```diff
@@ -29,4 +29,48 @@ Trello
...
+```
+3. Install the dependencies from the Gem file:\
+`bundle install`
+4. Set up the postgres DB:\
+`rake db:setup`
+5. Start up the app:\
+`rails s`
+
+### Troublshooting
+Some issues can arrise if you have different versions of Ruby installed or if you previously weren't using `rbenv`. Some helpful troubleshooting steps are:
```
Comment (jrapala) [inline README.md]: Typo: `arrise` should be `arise`

File: README.md [hunk 2]
```diff
@@ -29,4 +29,48 @@ Trello
...
+### Getting Started
+1. Download the repo `git clone git@github.com:Code-For-Chicago/Voma-backend.git`
+2. Make sure the correct version of Ruby is installed using `rbenv`:
+```
+rbenv init
+rbenv install 2.7.1
+rbenv rehash
+```
+3. Install the dependencies from the Gem file:\
+`bundle install`
```
Comment (jrapala) [inline README.md:59]: We should probably include instructions on how to install `bundler` as well.

File: README.md [hunk 3]
```diff
@@ -29,4 +29,48 @@ Trello
...
+4. Set up the postgres DB:\
+`rake db:setup`
+5. Start up the app:\
+`rails s`
+
+### Troublshooting
```
Comment (jrapala) [inline README.md]: Typo: `Troublshooting` should be `Troubleshooting

**Review by jrapala — CHANGES_REQUESTED** [review]
Just a couple of small comments

**Comment by cgundy** [comment]
> Does `brew` also start up a database instance for postgres related commands? I can make a small writeup on using the postgres commands directly for starting a dedicated project-specific instance for anyone who has installed postgres using yum/apt or compiled postgres from source.

I believe `rake db:setup` starts up the db. But yeah having more instructions would be very helpful!

**Comment by aaronreba** [comment]
> > Does `brew` also start up a database instance for postgres related commands? I can make a small writeup on using the postgres commands directly for starting a dedicated project-specific instance for anyone who has installed postgres using yum/apt or compiled postgres from source.
> 
> I believe `rake db:setup` starts up the db. But yeah having more instructions would be very helpful!

I've added a gist for starting a postgres database server with the cases I've thought of. https://gist.github.com/aaronreba/ce9619a8270e64cafa630d826d888092

**Comment by cgundy** [comment]
@jrapala is this PR good to merge?

**Pull request merged**